rocky/rocky/views/mixins.py

```diff
@@ -261,28 +261,6 @@ def __getitem__(self, key: Union[int, slice]) -> List[HydratedFinding]:
         raise NotImplementedError("FindingList only supports slicing")
 
 
-_EXCLUDED_OOI_TYPES = ("Finding", "FindingType")
-
-
-class MultipleOOIMixin(OctopoesView):
```

Comment:
This is unused (now)?

Reply:
Yes, it is not needed anymore as we can directly do an api call